### PR TITLE
Fixes : #8732 Support displayName field in at Tags 

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

Fixes : [#8732](https://github.com/open-metadata/OpenMetadata/issues/8732)

- Added  DisplayName field in the tags table 
- Added DisplayName Input field while creating tag and Classification.
- Converted the Old Form used for classification and tags to Antd form
- Adding Rename option in edit form
- Disable name edit for system level tags

image
image
image

image
image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] My PR title is `Fixes <issue-number>: <short explanation>`
- [x] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by chirag-madlani — COMMENTED** [review]
@Ashish8689 Let's remove rename as separate action and have it in Edit action only

**Comment by Ashish8689** [comment]
@chirag-madlani  changes updated as per comments!

File: openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                      t('message.entity-already-exists', {
+                        entity: t('label.name'),
+                      })
+                    );
+                  }
+                }
+
+                return Promise.resolve();
+              },
+            },
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/tags/TagsForm.tsx:120]: pull this out as separate function

**Comment by chirag-madlani** [comment]
@harshach can you review this to add support for displayName in tags & classifications

**Comment by harshach** [comment]
@Ashish8689 Display Name is optional

**Comment by Ashish8689** [comment]
@chirag-madlani  @harshach  updated the PR as per comments!

**Review by harshach — DISMISSED** [review]

**Pull request merged**